plugins/listblock/plugin.js

```diff
@@ -121,7 +121,7 @@ CKEDITOR.plugins.add( 'listblock', {
 
 				commit: function() {
 					this._.close();
-					this.element.appendHtml( this._.pendingHtml.join( '' ) );
+					this.element.appendHtml( CKEDITOR.tools.htmlSafeByReview(this._.pendingHtml.join( '' ), 'Every piece of pendingHtml is created using a safe template using safe internal values') );
```

Comment:
What is supposed to be in `pendingHtml` ?

Reply:
Looks like it would be a list of items created using template.output so I assume we would want to legacy convert this as well.

Reply:
Legacy converted